spec/requests/partner_users_requests_spec.rb

```diff
@@ -72,6 +73,8 @@
     end
 
     it "redirects back with alert if the user role removal fails" do
+      user.add_role(:org_admin)
```

Comment:
This line shouldn't be necessary since it's in a `before`block, no?

Reply:
Right. I think I missed this one. Fixed it here [f0ab010](https://github.com/rubyforgood/human-essentials/pull/4468/commits/f0ab01047271f26629c8c7c2363e38c75dbbcb47)